docs/devstack/README.md

```diff
@@ -76,6 +76,26 @@ To boot `devstack` on SLES 12 SP3 using modified versions of
         --parameter devstack_branch=sles-support \
         $USER-devstack-sles12-sp3
 
+You can also set the `devstack_extra_config` parameter to pass extra
+config to be placed in devstack's `local.conf`.  This can be done
+similarly to above via `--parameter devstack_extra_config=...`, but if
+the desired extra config is multi-line, it's slightly cleaner to do it
+by first creating [an environment
+file](https://docs.openstack.org/heat/rocky/template_guide/environment.html),
+e.g.:
+
+    cat <<EOF >devstack-branch-env.yaml
+    parameters:
+      devstack_extra_config: |
+        NOVA_REPO=https://review.openstack.org/p/openstack/nova
```

Comment:
nit: All the rest of the QA script isn't exporting variables, so it wouldn't make sense to write export NOVA_REPO... here. But I don't know why, I can't shake the feeling it would look more understandable... it's not really a config, but more extra shell things... Naming is hard...

Reply:
No - this really *is* config :-) It's not shell but rather config to be embedded in `local.conf`.

Reply:
It is not a question of naming, but if a process started from there should inherit these in its environment. Or here: What does devstack need in local.conf.